### PR TITLE
Fix rpmbuild on F33 and add it to schutzbot

### PR DESCRIPTION
**Pull request by ondrejbudai**
I decided to begin splitting my huge F33 PR into smaller chunks. This way it will be easier to review.

Let's start with fixing the build. This contains an ugly hack. Suggestions to make it nicer are welcome! See the commit messages for details.

### CODE REVIEW

File: schutzbot/mockbuild.sh [hunk 1]
```diff
@@ -27,13 +27,7 @@ fi
...
 # Install requirements for building RPMs in mock.
 greenprint "📦 Installing mock requirements"
-sudo dnf -y install createrepo_c make mock python3-pip rpm-build
-
-# Install s3cmd if it is not present.
-if ! s3cmd --version > /dev/null 2>&1; then
-    greenprint "📦 Installing s3cmd"
-    sudo pip3 -q install s3cmd
-fi
+sudo dnf -y install createrepo_c make mock rpm-build s3cmd
```
Comment (msehnout) [inline schutzbot/mockbuild.sh:30]: Is this change related to this PR?

**Review by msehnout — DISMISSED** [review]
Very nice!

File: schutzbot/mockbuild.sh (hunk 1 above)
Comment (ondrejbudai) [reply schutzbot/mockbuild.sh:30]: I forgot to include the explanation in the commit message while rebasing. Thanks for reminding me! I've just force-pushed an explanation, there's a copy:

s3cmd from pip doesn't work with Python 3.9. Fortunately, s3cmd in Fedora has a downstream patch to fix it.

**Review by msehnout — APPROVED** [review]
Thanks! CI is as usual: #1032
